views/blocks/profiling_tags.tpl

```diff
@@ -11,6 +11,26 @@
 
 [{if $oPayment->isCustomPaymentMethod()}]
 
+  [{if $oPayment->oxpayments__oxid->value === 'wdratepay-invoice'}]
+
+  [{assign var="sDeviceIdentToken" value=$oViewConf->getRatepayUniqueToken()}]
+  <script language="JavaScript">
+    var di = {t:"[{$sDeviceIdentToken}]",v:"WDWL",l:"Checkout"};
+  </script>
+  <script type="text/javascript" src="//d.ratepay.com/[{$sDeviceIdentToken}]/di.js">
+  </script>
+  <noscript>
+    <link rel="stylesheet" type="text/css"
+          href="//d.ratepay.com/di.css?t=[{$sDeviceIdentToken}]&v=WDWL&l=Checkout">
+  </noscript>
+  <object type="application/x-shockwave-flash" data="//d.ratepay.com/[{$sDeviceIdentToken}]/c.swf" width="0"
```

Comment:
I think we can safely ignore this Flash fallback, even for OXID. Most modern browsers stopped supporting Flash long ago and [Adobe is going to finally pull the plug next year](https://theblog.adobe.com/adobe-flash-update/).

Reply:
@tomazpu Any objections?

Reply:
I understand, but as it is officially documented with this fallback please leave it in.

Reply:
👍